### ISSUE TITLE
How are Slack configuration options supposed to be used?

### ISSUE THREAD

**Issue by GregorVoelkl** [spatie/laravel-backup#903]
In the configuration file, we have the option to specify the slack channel, as well as username and icon. 

The Slack documentation however says [here](https://api.slack.com/incoming-webhooks#posting_with_webhooks):

> 
You cannot override the default channel (chosen by the user who installed your app), username, or icon when you're using Incoming Webhooks to post messages. Instead, these values will always inherit from the associated Slack app configuration.

Do I miss something or are this configuration options legacy?

Thanks for this awesome package in either way!

**Comment by freekmurze** [comment]
I'm thinking you can override that, at least that worked for me in the past.

Our package simply defers to Laravel's native Slack functionality: https://laravel.com/docs/5.8/notifications#slack-notifications

**Comment by GregorVoelkl** [comment]
I have researched a bit. It seems like it is possible with [legacy integrations](https://api.slack.com/custom-integrations/incoming-webhooks)

With newly created apps changing this values for incoming webhooks is no longer possible via api call. This is also true for the https://github.com/laravel/slack-notification-channel package. I have just tested this.

Perhaps this could be mentioned in the configuration comment for this values.
